src/data/restore.ts

```diff
@@ -57,6 +71,19 @@ export const restore = (
         element.points = points;
       } else {
         if (isTextElement(element)) {
+          if ("font" in element) {
```

Comment:
let's delete this field off of the `element` also

Reply:
right, must have forgot :)